tests/FakeItEasy.Tests/Creation/CastleDynamicProxy/CastleDynamicProxyGeneratorTests.cs

```diff
@@ -215,11 +220,11 @@ public void Should_specify_that_private_class_was_not_found()
             // Arrange
 
             // Act
-            var type = Type.GetType("System.AppDomainInitializerInfo, mscorlib");
+            var type = Type.GetType("FluentAssertions.Common.NullReflector, FluentAssertions.Core");

```

Comment:
Good find, @jeremymeng. You'd asked if I'd tried on .netcore, and I hadn't. I'd really just eyeballed it. I was at work and don't have proper tools, and only limited time while my C++ compiles…